configs/config.toml.SAMPLE

```diff
@@ -108,4 +108,9 @@
 # The port on which the `/diagnostics` endpoint will be available can be
 # customized below.
 # [Diagnostics]
-	# Port = 8081
\ No newline at end of file
+	# Port = 8081
+
+[Actions]
```

Comment:
What do you think about naming it `[Applications]`? It's a little less cryptic to me than `[Actions]` and we'll be grouping configurations under application name (`[TBTC]`).

Reply:
I think `[Applications]` can be confusing because we have already `[SanctionedApplications]` section. What do you think about `[ApplicationActions]`?

Reply:
Maybe `ApplicationExtensions`? My reasoning is: we are _extending_ ECDSA client capabilities with whatever application-specific module wants to do. And those extensions will be eventually pluggable.

Reply:
Yeah makes sense! I'll change it.